src/deploy.py

```diff
@@ -77,13 +77,13 @@ def configure_montagu(is_first_time, settings):
         data_import.do(settings)
     orderly.configure_orderly(not data_exists, settings)
 
-    passwords = database.setup(settings["use_real_passwords"])
+    passwords = database.setup(settings["password_group"])
 
     # Push secrets into containers
     cert_paths = get_ssl_certificate(settings["certificate"])
     token_keypair_paths = get_token_keypair()
 
-    configure_api(passwords['api'], token_keypair_paths, settings["hostname"], settings["use_real_passwords"])
+    configure_api(passwords['api'], token_keypair_paths, settings["hostname"], settings["password_group"] == 'production')
```

Comment:
This code would be clearer if we put that last argument in a variable - just so it's clear what its purpose is. e.g.

```
send_emails = settings["password_group"] == 'production'
configure_api(passwords['api'], token_keypair_paths, settings["hostname"], send_emails)
```